packages/client/hmi-client/src/components/widgets/tera-common-modal-dialogs.vue

```diff
@@ -96,7 +96,8 @@ watch(
 	() => isCopyDialogVisible.value,
 	async () => {
 		if (isCopyDialogVisible.value) {
-			const name = await useProjects().getActiveProjectName();
+			project.value = useProjects().activeProject.value;
```

Comment:
What is this line for?

Reply:
There were two issues in this code. The first was where the copy happens we use `project.value` but it was never set (a PR in january took away this line and broke it). The second is on the front page we didn't set the project clicked on and were only looking at the active project.